### ISSUE TITLE
Sprint goal week 5 -- refinement, components, and styling

### ISSUE THREAD

**Issue by Kalle-Rei** [DenizeScheffold/ArcticWell#51]
Continue work on the migration to Next.js/React. In particular components like the navbar and header should be polished and fully implemented across all relevant pages. 

The map view should be styled according to the style guide. Restoring previous functionality (unique icons and info about the nearest box) is __not__ part of the sprint goal, but could potentially be done. 

Preparations should also be made for future implementations of a weather API and for adding more functions from the Google Maps API to the map view.

If there is time, empty pages that follow the structure of the site map can be created, and components added to them.

---

To summarize:
1. components such as the navbar should be fully implemented
2. preparations for future use of APIs should be made
